data/campaigns/Under_the_Burning_Suns/scenarios/01_The_Morning_After.cfg

```diff
@@ -752,6 +755,12 @@
         # Lake monster event
```

Comment:
Having reread the discussion about XP, I agree with skipping the lake monster if the player has swum through the lake to get Zhul earlier; rather than a reward it's a balance for the fact that they're up 3 units.

However, please add a comment about that here.